packages/react-native-tab-view/package.json

```diff
@@ -43,10 +43,9 @@
   },
   "devDependencies": {
     "del-cli": "^5.0.0",
-    "react": "18.1.0",
```

Comment:
Was this on purpose? I'm not sure if we'd want to remove `react` from our dependencies

Reply:
It was removed by mistake. I've fixed it.